config.yaml.example

```diff
@@ -1,6 +1,13 @@
 ---
 jenkins_url: https://myjenkins.ourcorporate.com
 job_search_fields: my-job-prefix-or-regex
+stage_log:
```

Comment:
If we need this field, let's put it at the bottom of the config

Reply:
Done